.github/workflows/test-ske.yaml

```diff
@@ -1,4 +1,4 @@
-name: test-ske
+name: SKE Tests
```

Comment:
I think you need to update: https://github.com/syntasso/enterprise-kratix/blob/main/.github/workflows/trigger-ci.yaml#L31

Reply:
good shout, i'll actually revert the name changes as to not bundle the two changes (this was mostly for readability :p)